rocky/rocky/views/organization_member_add.py

```diff
@@ -30,7 +30,7 @@ def get_form_kwargs(self):
         return kwargs
 
     def get_success_url(self, **kwargs):
-        return reverse_lazy("organization_detail", kwargs={"organization_code": self.organization.code})
+        return reverse_lazy("organization_settings", kwargs={"organization_code": self.organization.code})
```

Comment:
I'd expect a redirect to the `organization_member_list` page, as that is where the 'Add new member' button is.

Reply:
Fair point. Before it was part of the detail view, which got renamed. Didn't double check the occurrences , but updated it now.